### PR TITLE
Feature - Recurring Events + Tickets

### PR DESCRIPTION
**Pull request by lucatume**

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Tribe__Tickets__Main {
 	/**
 	 * Current version of this plugin
 	 */
-	const VERSION = '5.5.9.1';
+	const VERSION = '6.0.0-dev';
```
Comment (juanfra) [inline src/Tribe/Main.php:11]:
```suggestion
	const VERSION = '5.6.0-dev';
```

File: tests/integration/TEC/Tickets/Emails/Email_HandlerTest.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	 * It should create tickets emails posts correctly
+	 *
+	 * @test
+	 */
+	public function should_create_tickets_emails_posts_correctly(): void {
+		$handler = new Email_Handler( tribe() );
+
+		// Filter the emails to return the default ones.
+		$defaul_emails = $handler->get_default_emails();
+		add_filter( 'tec_ticketk_emails_registered_emails', static fn() => $defaul_emails );
```
Comment (juanfra) [inline tests/integration/TEC/Tickets/Emails/Email_HandlerTest.php:53]:
```suggestion
		add_filter( 'tec_tickets_emails_registered_emails', static fn() => $defaul_emails );
```

File: tests/integration/TEC/Tickets/Emails/Email_HandlerTest.php [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+	 * It should report 0 if there are no emails to create
+	 *
+	 * @test
+	 */
+	public function should_report_0_if_there_are_no_emails_to_create(): void {
+		$handler = new Email_Handler( tribe() );
+
+		// Filter the emails to return an empty array: no email to create.
+		$defaul_emails = $handler->get_default_emails();
+		add_filter( 'tec_ticketk_emails_registered_emails', '__return_empty_array' );
```
Comment (juanfra) [inline tests/integration/TEC/Tickets/Emails/Email_HandlerTest.php:68]:
```suggestion
		add_filter( 'tec_tickets_emails_registered_emails', '__return_empty_array' );
```

File: tests/integration/TEC/Tickets/Emails/Email_HandlerTest.php [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+				return new \WP_Error( 'For reasons' );
+			}
+
+			return wp_insert_post( $args );
+		}, true );
+		$handler = new Email_Handler( tribe() );
+
+		// Filter the emails to return an empty array: no email to create.
+		$defaul_emails = $handler->get_default_emails();
+		add_filter( 'tec_ticketk_emails_registered_emails', static fn() => $defaul_emails );
```
Comment (juanfra) [inline tests/integration/TEC/Tickets/Emails/Email_HandlerTest.php:92]:
```suggestion
		add_filter( 'tec_tickets_emails_registered_emails', static fn() => $defaul_emails );
```

File: src/Tickets/Commerce/Module.php [hunk 5]
```diff
@@ -594,7 +594,7 @@ public function save_ticket( $post_id, $ticket, $raw_data = [] ) {
 	 *
 	 * @since 5.1.9
 	 *
-	 * @since 5.5.10 Adjust the method to handle both Ticket and Attendee post type deletion separately.
+	 * @since TBD Adjust the method to handle both Ticket and Attendee post type deletion separately.
```
Comment (Camwyn) [inline src/Tickets/Commerce/Module.php:597]: Why is this TBD now?

File: src/Tickets/Commerce/Ticket.php [hunk 6]
```diff
@@ -491,6 +499,7 @@ public function save( $post_id, $ticket, $raw_data = [] ) {
 		}
 
 		update_post_meta( $ticket->ID, '_price', $ticket->price );
+		update_post_meta( $ticket->ID, '_type', $ticket->type() ?? 'default' );
```
Comment (Camwyn) [inline src/Tickets/Commerce/Ticket.php:502]: Should this use `$type_meta_key` from above?
(Ditto price, etc, but OOS)

**Review by Camwyn — COMMENTED** [review]
Looking good, didn't get 100% but found a couple little things so commenting until I can get back

**Pull request merged**